cfn-resources/federated-database-instance/cmd/resource/resource.go

```diff
@@ -28,13 +28,12 @@ import (
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
-	"github.com/spf13/cast"
-	atlasSDK "go.mongodb.org/atlas-sdk/v20231115002/admin"
+	atlasSDK "go.mongodb.org/atlas-sdk/v20231115008/admin"
 )
 
-var CreateRequiredFields = []string{constants.ProjectID, constants.TenantName, constants.DataFederationRoleID, constants.DataFederationTestS3Bucket, constants.DataProcessRegion}
+var CreateRequiredFields = []string{constants.ProjectID, constants.TenantName}
 var ReadRequiredFields = []string{constants.ProjectID, constants.TenantName}
-var UpdateRequiredFields = []string{constants.ProjectID, constants.TenantName, constants.DataFederationRoleID, constants.DataFederationTestS3Bucket, constants.SkipRoleValidation}
+var UpdateRequiredFields = []string{constants.ProjectID, constants.TenantName}
```

Comment:
you can have just one var CreateReadUpdateDeleteRequiredFields, or just call it RequiredFields, apart from ListRequiredFields

Reply:
I thought about simplifying this but I prefer to keep it as it is for:
- Consistency with other resources
- Possible future evolution of required fields
- You can see very quickly which are the required fields for each operation, even if they are the same
- Not a fan of `CreateReadUpdateDeleteRequiredFields ` 😅